openfermioncirq/gates/four_qubit_gates_test.py

```diff
@@ -56,63 +54,55 @@ def test_state_swap_eigen_component(index_pair, n_qubits):
         assert numpy.allclose(actual_component, expected_component)
 
 
-def test_double_excitation_repr():
```

Comment:
Where are `DoubleExcitationGate.__repr__` and `CombinedDoubleExcitationGate.__repr__` tested?

Reply:
In the call `ofc.testing.assert_implements_consistent_protocols(ofc.DoubleExcitation)` and `ofc.testing.assert_implements_consistent_protocols(ofc.CombinedDoubleExcitationGate())`.

Reply:
:+1: That's a great helper function.